src/components/encounter/ImageBadge.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+
+import { colors, device } from '../../config';
+import { Image } from '../Image';
+
+type Props = {
+  verified: boolean;
+};
+
+export const ImageBadge = ({ verified }: Props) => {
+  return (
+    <View style={styles.badge}>
+      <Image
+        // TODO: add proper badges
```

Comment:
was this TODO part of the task here and should be done?

Reply:
this should be handled in #333